### PR TITLE
Upmerge 08 03 2022

### PR DESCRIPTION
**Pull request by nordicjm**

### CODE REVIEW

File: nfc/CHANGELOG.rst [hunk 1]
```diff
@@ -9,6 +9,14 @@ Changelog
 
 All notable changes to this project are documented in this file.
 
+master
+******
```
Comment (umapraseeda) [inline nfc/CHANGELOG.rst:13]:
```suggestion
Main branch
***********
```

Comment (nordicjm) [reply nfc/CHANGELOG.rst:13]: @maje-emb can you fix this and send another PR?

**Review by frkv — APPROVED** [review]
LGTM (Crypto)

**Review by czeslawmakarski — APPROVED** [review]
LGTM for 802.15.4

**Pull request merged**